src/common/wizard.js

```diff
@@ -0,0 +1,64 @@
+import _ from 'messages'
+import classNames from 'classnames'
+import every from 'lodash/every'
+import Icon from 'icon'
+import map from 'lodash/map'
+import React, { Component, cloneElement } from 'react'
+
+import styles from './wizard.css'
+
+const Wizard = ({ children }) => {
+  const allDone = every(React.Children.toArray(children), (child) => child.props.done || child.props.summary)
+  return <ul className={styles.wizard}>
+    {map(children, (child, key) => cloneElement(child, { allDone, key }))}
+  </ul>
+}
+export { Wizard as default }
+
+export class Section extends Component {
+  componentWillMount () {
+    this.setState({isActive: false})
+  }
+  updateFocus = (event) => {
+    const section = event.currentTarget
+    this.setState({isActive: section.contains(section.ownerDocument.activeElement)})
```

Comment:
I don't like it very much because you are bypassing React.

I am not opposed but prefer React equivalents when possible otherwise add a comment explaining what you are doing.